requirements.txt

```diff
@@ -5,4 +5,5 @@ scipy
 pandas==1.3.5
 shapely
 rasterio
-xarray
\ No newline at end of file
+xarray
+requests-mock
```

Comment:
Remove from requirements.txt, to keep only in requirements-dev.txt

Reply:
Done!